src/main/java/duke/Gui.java

```diff
@@ -0,0 +1,234 @@
+package duke;
+
+import java.time.LocalDate;
+
+import duke.exception.DukeStorageException;
```

Comment:
that's a lot of import statements! maybe it will look nicer if you group them accordingly?